packages/web-components/src/components/va-banner/va-banner.tsx

```diff
@@ -186,6 +186,7 @@ export class VaBanner {
     return (
       <Host>
         <va-alert
+          id="va-banner-alert"
```

Comment:
This `id` has been added to help us identify easier when `va-alert` component is being used in the `va-banner`. In this scenario, we are not applying the slim prop.